dali/ft/filecopy.cpp

```diff
@@ -2991,7 +2993,7 @@ void FileSprayer::updateTargetProperties()
             offset_t physPartLength = curProgress.outputLength;
             if (copyCompressed) {
                 FilePartInfo & curSource = sources.item(cur.whichInput);
-                partLength = curSource.size;
+                partLength = physPartLength;
```

Comment:
physPartLength (curProgress.outputLength) is the uncompressed part size?

vs curProgress.compressedPartSize, which is the compressed part size.

If that's correct, I think that physPartLength variable should be renamed, as a little confusing at the moment. i.e. it is not the 'physical part length' if compressed.